packager.sh

```diff
@@ -56,25 +90,9 @@ sig_hand () {
     echo "signal trapped"
     if [ "$lxc_name" != "" ]; then
         set +e
+        scp "${lxc_ip}:ssh.log" "out_${BUILD_UBUVER}/ssh.history"
```

Comment:
`BUILD_UBUVER` is set here: https://github.com/gem/oq-hazardlib/blob/b2fbe91ec9fe243882e7410db9cd409fa490f1c1/packager.sh#L595